cpu/stm32/include/candev_stm32.h

```diff
@@ -100,6 +100,7 @@ typedef struct {
 #ifndef CPU_FAM_STM32F1
     gpio_af_t af;               /**< Alternate pin function to use */
 #endif
+    bool en_deep_sleep_wake_up; /**< Enable deep-sleep wake-up interrupt */
```

Comment:
Why not extend the bit field below? (line 130)

Reply:
The bitfield was using `uint8_t` as the base type and there were already eight bits in it. Also, the bits in the bit field correspond to a setting in the config register.